operator/config/olm-catalog/ibm-spectrum-scale-csi-operator/manifests/ibm-spectrum-scale-csi-operator.clusterserviceversion.yaml

```diff
@@ -553,4 +553,4 @@ spec:
   provider:
     name: IBM
   replaces: ibm-spectrum-scale-csi-operator.v2.4.0
```

Comment:
This should be replacing v2.5.0. Similar change needed in other CSVs

Reply:
Fixed.